specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -3186,6 +3186,12 @@
           }
         ],
         "responses": {
+          "200": {
```

Comment:
Are these swagger bugfixes to bring swagger in-line with how the API has always behaved?  If not, these should go into a new api-version as the existing preview version is almost 2 years old.

Reply:
Yes, this is a bug fix. It exists in production for more then half a year and has no costumer impact (not breaking any scenario). We are aware that in the stable version this kind of change will require a new version.